mapic/mistapiconnector_app.go

```diff
@@ -810,6 +815,18 @@ func (mc *mac) refreshStreamInfo(playbackID string) (*streamInfo, error) {
 	return info, nil
 }
 
+func (mc *mac) GetCachedStream(playbackID string) *api.Stream {
```

Comment:
An alternative to the refactor I proposed would be exposing the `getStreamInfo` function as this `GetCachedStream` (without `Cached` I guess), then the actual `log_ext_data.go` would only need to handle assets.

Still optional though, just thinking how to keep the "studio is just a provider" reality a lil closer.

Reply:
Yeah, let's maybe leave thinking about this more if we decide to have "studio as a provider". This part will definitely not make it harder 🙃